package.json

```diff
@@ -189,6 +190,9 @@
         "statements": 86.6
       }
     },
+    "moduleNameMapper": {
+      "^styled-components": "styled-components/dist/styled-components.browser.cjs.js"
```

Comment:
I forget why we needed this -- is it documented anywhere why we have this? Would be helpful to keep note for future reference.

Reply:
I believe we were having issues with the jest tests, the suggestion to add this line came from https://github.com/styled-components/jest-styled-components/issues/144#issuecomment-394184823